src/components/cards/WorkTimerSetup.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import Button from "../../components/bites/Button";
+import CronometerTimeView from "../../utils/CronometerTimeView";
+
+function WorkTimerSetup({ setWorkTimeButton, workTime }) {
+  return (
+    <div className="WorkTime">
+      <h3 className="text-3xl font-bold p-2">Work Time</h3>
+      <div className="flex justify-center items-center">
+        <div className="flex flex-col col-25">
+          <Button onClickFn={setWorkTimeButton} onClickParams={-1} text="-0.1s" color="red" />
+          <Button onClickFn={setWorkTimeButton} onClickParams={-10} text="-1s" color="red" />
```

Comment:
This multiples buttons declarations, could be an array, coverted 'onClickParams' and 'text' to object params

Reply:
Updated in 834ebe8